src/views/partials/list-settings.handlebars

```diff
@@ -0,0 +1,25 @@
+<div id="listSettingsModal" class="modal">
+    <!-- Exit button (needs css) -->
+    <label for="Close List settings" class="close-btn">X</label>
+    
+    <!-- Form for changing the list -->
+    <form action="/list/{{boardId}}/{{list._id}}" method="POST">
+        <div class="listNameDiv">
+            <label for="List Name">Name: </label>
+            <input id="listName" name="listName" type="text" placeholder="list name" value="{{list.listName}}"/>
+        </div>
+        <div class="positionValueDiv">
+            <label for="Position">Position: </label>
+            <input id="positionValue" name="position" type="number" placeholder="position" value="{{list.position}}" min="1" max="{{positions.length}}"/>
+        </div>
+        <div class="submitButtonDiv">
+            <button type="submit">Submit</button>
+        </div>
+    </form>
+    <!--Form for deleting the list -->
+    <form action="delete/list/{{boardId}}/{{list._id}}" method="POST">
```

Comment:
This doesn't have to be a whole ass form, can just be a button. The button's on click function will make the ajax request in the client side. Look at my user page client side for reference (currently it's a pull request as of writing)

Reply:
Unnecessary change